pkg/partmgr/partmgr.go

```diff
@@ -169,7 +184,9 @@ func (p *PartitionManager) updatePartitionInSchema(partition *DBPartition) error
 	if err != nil {
 		return errors.Wrap(err, "Failed to update new partition in schema file")
 	}
-	err = p.container.Sync.PutObject(&v3io.PutObjectInput{Path: path.Join(p.path, config.SCHEMA_CONFIG), Body: data})
+	if p.container != nil {
+		err = p.container.Sync.PutObject(&v3io.PutObjectInput{Path: path.Join(p.path, config.SCHEMA_CONFIG), Body: data})
+	}
```

Comment:
else what?

Reply:
else nothing, file will not be updated (it can be nil only in tests)

Reply:
it seems to me that `nil` container is an error rather than valid state